crates/aptos-rosetta/src/account.rs

```diff
@@ -136,55 +136,53 @@ async fn get_balances(
                     }
 
                     let store: Store = bcs::from_bytes(&bytes)?;
-                    if account.is_total_stake() {
-                        // For total stake, collect all underlying staking contracts and combine
-                        let mut total_stake: Option<u64> = None;
-                        maybe_operators = Some(vec![]);
-                        for (operator, contract) in store.staking_contracts {
-                            // Keep track of operators
-                            maybe_operators.as_mut().unwrap().push(operator);
-                            match get_total_stake(
+                    let mut total_requested_balance: Option<u64> = None;
+                    maybe_operators = Some(vec![]);
```

Comment:
It also may be cleaner to do use the operators directly so you don't have to do the `as_mut().unwrap().push(operator)`

```
let mut operators = vec![];

...
operators.push(operator);

...

maybe_operators = Some(operators)
```